config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
   resources :spaces do
-    resources :reservations, only: [:new, :create]
+    resources :reservations, only: [:new, :create, :index, :show]
   end
+  get "reservations", to: 'reservations#index'
```

Comment:
duplicate

Reply:
TA said I have to write it outside as well. He dictated it haha. But to be honest that guy had no idea.

Reply:
ah sorry makes sense.. keep it that way 👍